docs/README.md

```diff
@@ -21,11 +21,22 @@ the following Parallels builders:
   VM export you want to use as the source. As an additional benefit, you can
   feed the artifact of this builder back into itself to iterate on a machine.
 
-## Requirements
+- [parallels-ipsw](/packer/plugins/builders/parallels/ipsw) - Starts from an IPSW
+  file, creates a brand new Parallels Mac OS VM, installs an OS, provisions software
+  within the OS, then exports that machine to create an image. This is best
+  for people who want to start from scratch.
+
+- [parallels-macvm](/packer/plugins/builders/parallels/macvm) - This builder imports
```

Comment:
URL is different compared to existing builders. refer [this](https://github.com/Parallels/packer-plugin-parallels/pull/92/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R18)

Reply:
Fixed. I see the existing ones also wrong, fixed it.